src/analysisd/decoders/dbsync.c

```diff
@@ -31,7 +31,7 @@ extern void mock_assert(const int result, const char* const expression,
 static void dispatch_send_local(dbsync_context_t * ctx, const char * query) {
     int sock;
 
-    if (strcmp(ctx->component, "syscheck") == 0) {
+    if (strcmp(ctx->component, "syscheck") == 0 || strncmp(ctx->component, "syscollector", 12) == 0) {
```

Comment:
I'd suggest using the same secure function for "syscheck" too. `strncmp` (`strncmp(ctx->component, "syscheck", 8)`)